src/igvfd/schemas/publication.json

```diff
@@ -0,0 +1,131 @@
+{
+    "title": "Publication",
+    "description": "Schema for a publication object.",
+    "$id": "/profiles/publication.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "title",
+        "award",
+        "lab",
+        "identifiers"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "title"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/attachment"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "title": {
+            "title": "Title",
+            "description": "Title of the publication or communication.",
+            "uniqueKey": true,
+            "type": "string"
+        },
+        "abstract": {
+            "title": "Abstract",
+            "description": "Abstract of the publication or communication.",
+            "type": "string"
+        },
+        "authors": {
+            "title": "Authors",
+            "type": "string"
+        },
+        "date_published": {
+            "title": "Publication date",
+            "description": "The date the publication or communication was published; must be in YYYY-MM-DD format.",
+            "type": "string",
+            "oneOf": [
+                {
+                    "format": "date"
+                },
+                {
+                    "comment": "Historical date formats, not valid for new submissions.",
+                    "permission": "import_items",
+                    "pattern": "^[0-9]{4}[^-]*$"
+                }
+            ]
+        },
+        "date_revised": {
```

Comment:
we discussed it at a wranglers meeting, do not use  date-time for any of the date properties

only timestamps are date-time